data/migrations/20.lua

```diff
@@ -0,0 +1,3 @@
+﻿function onUpdateDatabase()
+	return false
+end
```

Comment:
Thanks, I'd like you to make a pr (in my repo) fixing thus as I am not able now.